### PR TITLE
ToT to Edep general converter

### PR DESCRIPTION
**Pull request by wkrzemien**
Add ToT to Edep converter with a kind of lookup table.
Some tests are added but not full list.
This PR should be accepted after JPetCachedFunction PR from j-pet-framework core.

### CODE REVIEW

File: LargeBarrelAnalysis/ToTConverter.cpp [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+ToTConverter::ToTConverter(const ToTConverterParams& params): fParams(params)
+{
+  TFormula func("myFunc", fParams.fFormula.c_str());
+  func.SetParameters(fParams.fParams.data());
+  if (fParams.fBins <= 0) {
+    ERROR("Number of bins must be greater than 0! getEdep() function will not work correctly");
+    fParams.fValidFunction = false;
+    return;
+  }
+  double step = (fParams.fEdepMax - fParams.fEdepMin) / fParams.fBins;
```
Comment (grey277) [inline LargeBarrelAnalysis/ToTConverter.cpp]: I think step can be extracted as class variable(it is also used in edepToIndex)

**Comment by alekgajos** [comment]
Please retest this.

**Comment by JPETTomographyTester** [comment]
Found 1 violations:

**Reporter**: root-6-cppcheck
**Rule**: unusedFunction
**Severity**: INFO
**File**: /var/jenkins_home/workspace/j-pet-framework-examples-PRs-root-6/LargeBarrelAnalysis/ToTConverter.cpp L20

The function &#39;getConverter&#39; is never used.


*<this is a auto generated comment from violation-comments-lib F7F8ASD8123FSDF>* *<a-629430137>*
 *<ACCUMULATED-VIOLATIONS>*

**Comment by JPETTomographyTester** [comment]
Found 1 violations:

**Reporter**: root-5-cppcheck
**Rule**: unusedFunction
**Severity**: INFO
**File**: /var/jenkins_home/workspace/j-pet-framework-examples-PRs-root-5/LargeBarrelAnalysis/ToTConverter.cpp L20

The function &#39;getConverter&#39; is never used.


*<this is a auto generated comment from violation-comments-lib F7F8ASD8123FSDF>* *<a2120276847>*
 *<ACCUMULATED-VIOLATIONS>*

**Comment by JPETTomographyTester** [comment]
Found 1 violations:

**Reporter**: root-6-cppcheck
**Rule**: unusedFunction
**Severity**: INFO
**File**: /var/jenkins_home/workspace/j-pet-framework-examples-PRs-root-6/LargeBarrelAnalysis/ToTConverter.cpp L20

The function &#39;getConverter&#39; is never used.


*<this is a auto generated comment from violation-comments-lib F7F8ASD8123FSDF>* *<a-629430137>*
 *<ACCUMULATED-VIOLATIONS>*

**Comment by JPETTomographyTester** [comment]
Found 1 violations:

**Reporter**: root-5-cppcheck
**Rule**: unusedFunction
**Severity**: INFO
**File**: /var/jenkins_home/workspace/j-pet-framework-examples-PRs-root-5/LargeBarrelAnalysis/ToTConverter.cpp L20

The function &#39;getConverter&#39; is never used.


*<this is a auto generated comment from violation-comments-lib F7F8ASD8123FSDF>* *<a2120276847>*
 *<ACCUMULATED-VIOLATIONS>*

**Comment by JPETTomographyTester** [comment]
Found 1 violations:

**Reporter**: root-6-cppcheck
**Rule**: unusedFunction
**Severity**: INFO
**File**: /var/jenkins_home/workspace/j-pet-framework-examples-PRs-root-6/LargeBarrelAnalysis/ToTConverter.cpp L20

The function &#39;getConverter&#39; is never used.


*<this is a auto generated comment from violation-comments-lib F7F8ASD8123FSDF>* *<a-629430137>*
 *<ACCUMULATED-VIOLATIONS>*

**Comment by JPETTomographyTester** [comment]
Found 1 violations:

**Reporter**: root-5-cppcheck
**Rule**: unusedFunction
**Severity**: INFO
**File**: /var/jenkins_home/workspace/j-pet-framework-examples-PRs-root-5/LargeBarrelAnalysis/ToTConverter.cpp L20

The function &#39;getConverter&#39; is never used.


*<this is a auto generated comment from violation-comments-lib F7F8ASD8123FSDF>* *<a2120276847>*
 *<ACCUMULATED-VIOLATIONS>*

**Pull request merged**